src/Microsoft.VisualStudio.Editors/DesignerFramework/AccessModifierComboBox.vb

```diff
@@ -527,7 +527,7 @@ Namespace Microsoft.VisualStudio.Editors.DesignerFramework
             Catch ex As Exception When ReportWithoutCrash(ex, "Failed to determine if the access modifier combobox should be enabled", NameOf(AccessModifierCombobox))
                 Throw
             End Try
-            Return shouldBeEnabled
+            Return ShouldBeEnabled()
         End Function
```

Comment:
Note this "appears" like a behavior change, but its not. VB was already binding this to ShouldBeEnabled(). It's actually dead code.